src/S3ErgodicMoves.h

```diff
@@ -387,6 +387,33 @@ auto make_26_move(T1&& universe,
     return std::move(universe);
 }  // make_26_move()
 
+template <typename T1>
+auto try_62_move(T1&& universe_ptr, Vertex_handle candidate){
+    std::vector<Cell_handle> candidate_cells;
+    auto adjacent_cell = std::make_tuple(0, 0, 0); //holds how many (3, 1), (2, 2), and (1, 3) adjacent cells exist
+    (universe_ptr.triangulation)->incident_cells(candidate, back_inserter(candidate_cells));
+    for (auto cit: candidate_cells){
+      CGAL_triangulation_precondition(is_cell(cit));
```

Comment:
I get this error: 
/Users/adam/CDT-plusplus/src/S3ErgodicMoves.h:396:39: error: use of undeclared identifier 'is_cell'
      CGAL_triangulation_precondition(is_cell(cit));